schemes/enc/v1/README.md

```diff
@@ -0,0 +1,146 @@
+# Dapr encryption scheme v1: `dapr.io/enc/v1`
```

Comment:
Indeed, please use `encryption`.

Reply:
If this is not for human consumption, then why the nicely formatted domain? "Enc" is too ambiguous. It can mean encryption or encoding.

Reply:
> If this is not for human consumption, then why the nicely formatted domain? 

It is common pattern to make them URLs. For example, age, which was a big inspiration for this scheme, does that too: https://github.com/C2SP/C2SP/blob/main/age.md#header

The benefit is that it's a valid URL, so I've already opened a [PR in dapr/website](https://github.com/dapr/website/pull/76) to make that point to the README with the specs. If someone were to read that and had doubts on the meaning of "enc", they could visit the URL.

Reply:
> They're not meant for human consumption

Contributors consume this code when, don't they?

> For example, age, which was a big inspiration for this scheme, does that too: https://github.com/C2SP/C2SP/blob/main/age.md#header

Age is not our concern.

> so less bytes in the output files, the better.

The benefit of creating clarity far outweighs the negligible and vastly insignificant increase of 7 bytes. Had length been an issue I'd imagine you wouldn't have constructed this is a URL to begin with.

Reply:
As discussed on discord this will not change in this PR.

after the PR is merged, changes are welcome to the URL if anyone were up for that. They would likely require changes to Dapr/dapr, go-sdk, website, and possibly docs.

Reply:
We agreed to keep it as-is for now. We might decide to make it a breaking change in the future but is not a blocker for this PR.